src/textfield/textfield.js

```diff
@@ -213,12 +213,33 @@
    * @public
    */
   MaterialTextfield.prototype.change = function(value) {
-
     this.input_.value = value || '';
     this.updateClasses_();
   };
   MaterialTextfield.prototype['change'] = MaterialTextfield.prototype.change;
 
+   /**
+   * Focus text field.
```

Comment:
Sure.
I edited the file using the online editing tool and did the commit. It
should be fine now.

As a side note the call to this.updateClasses_(); is not necessary today
because
the blur and focus events are synchronous and these events are already
handled.

But who knows what tricks will perform this.updateClasses_(); in the future.

On Thu, Mar 3, 2016 at 2:45 PM, Sérgio Gomes <notifications@github.com>
wrote:

> In src/textfield/textfield.js
> <https://github.com/google/material-design-lite/pull/4168#discussion_r54880413>
> :
>
> >      this.input_.value = value || '';
> >      this.updateClasses_();
> >    };
> >    MaterialTextfield.prototype['change'] = MaterialTextfield.prototype.change;
> >
> > +   /**
> > +   * Focus text field.
>
> JSCS is being picky and complaining about the indenting in these comments.
> Could you change these to be:
>
> /** * Focus text field. * [...]
>
> instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/material-design-lite/pull/4168/files#r54880413>
> .
>